src/chem_parse/ChemlabParser.py

```diff
@@ -146,7 +147,8 @@ def p_built_in(self, p):
                 else:
                     print("----convert: " + str(p[3]) + " ; value: " + str(p[6]) + " from: " + str(
                         p[8]) + " ; to: " + str(p[10]))
-            p[0]["value"] = 24.6
+
+            p[0]["value"] = feature.convertTo(p[3],p[6],(p[8])[0],(p[10])[0])
```

Comment:
We have two rules that apply to conversion in the grammar. We are not taking it into account. We need to fix this

Reply:
Also where do you pass the prefix in here? Is it mixed with the unit?